docs/marketing/detailed-description.md

```diff
@@ -0,0 +1,18 @@
+The KOMOJU extension provides an easy way to accept Japanese payments for your Magento store. Credit Card payments only make up a small percentage of online payments in Japan, with KOMOJU you can accept payments in multiple ways, allowing your customers to pay in the way that suits them best.
+
+Note: Currently the extension only allows payments for stores with JPY as the base currency.
+
+## Account & Pricing
+
+A KOMOJU account is required to use this extension. If you do not have a KOMOJU account yet you will need to [sign up here](https://komoju.com/) for one before using the extension. See [this link](https://komoju.com/#pricing) for pricing details.
+
+## Security
+
+- PCI Compliant
+- No Credit Card details stored in Magento
```

Comment:
I think we can leave out the second point since its implied by the PCI compliance. Maybe we can remove this section and add something about PCI compliance under features

Reply:
Done. I've rolled it into ["Features"](https://github.com/komoju/komoju-magento/pull/4/commits/8ebe88d740fbd06bc9c8c1e64d55c506d41e492c):
```
PCI Compliant; We do the hard work of keeping your user's data secure so you don't need to worry.
```